modules/ROOT/pages/fd-iterating.adoc

```diff
@@ -0,0 +1,112 @@
+= Process Elements in a Collection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can process elements in a collection that is passed in a payload to an instance of the For-Each component or of the Parallel For-Each component. Each component sets a scope within which you can add other types of component, as well as connectors and modules, for processing each element.
+
+.The For-Each component and the Parallel For-Each component both set scopes for processing the elements within a collection.
+image::fd-iterators-example.png[Two simple examples: one of the use of For-Each, the other of the use of Parallel For-Each]
+
+<1> The For-Each component in this example inserts each element, one at a time, into a database. Each element is presumably a string that can be used in a SQL INSERT statement.
+<2> The Parallel For-Each component in this example sends each element to a Set Payload component that modifies the element with an expression and adds it to the flow's payload.
+
+Both types of component can process any collection. A collection can be any supported content type, such as application/json, application/java, or application/xml.
```

Comment:
I would remove the following:
```
 A collection can be any supported content type, such as application/json, application/java, or application/xml.
```
Because a collection doesn't have an application type per se.